velox/functions/prestosql/tests/RandTest.cpp

```diff
@@ -74,5 +93,146 @@ TEST_F(RandTest, nonNullInt8) {
   EXPECT_LT(rand(4), 4);
 }
 
+TEST_F(RandTest, zeroArgSecure) {
```

Comment:
I've generally seen test names given as functionName followed by argType. So secureRandZeroArg, secureRandInt32, secureRandInt64 etc. Please can you follow that convention.